### PR TITLE
Introducing markdown

### PR DESCRIPTION
**Pull request by JesperIRL**
Fixed a few bugs from html to markdown conversion.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * Iris Clark ([iris](@irisclark) - **Reviewer**) ⚠️ Review applies to 72ca84289ab02dd62ecc99ad1bf40c7c5ce4b6ec

### Download
`$ git fetch https://git.openjdk.java.net/guide pull/4/head:pull/4`
`$ git checkout pull/4`

### CODE REVIEW

File: .jcheck/conf [hunk 1]
```diff
@@ -22,7 +22,7 @@
 ;
 
 [general]
-project=guide
+project=jdk
```
Comment (rwestberg) [inline .jcheck/conf:25]: This change looks like an unintended merge, should probably revert it!

File: .jcheck/conf [hunk 2]
```diff
@@ -36,4 +36,4 @@ domain=openjdk.org
 files=.*\.java$|.*\.c$|.*\.h$|.*\.cpp$|.*\.hpp$
 
 [checks "reviewers"]
-reviewers=1
+committers=1
```
Comment (rwestberg) [inline .jcheck/conf:39]: Same here.

Comment (JesperIRL) [reply .jcheck/conf:39]: Interesting. The conf file wasn't even in the list of conflicting files.

**Pull request closed without merging**